v19.2/cockroachcloud-cluster-management.md

```diff
@@ -7,6 +7,31 @@ build_for: [cockroachcloud]
 
 This page describes the cluster management, backup/restore, and cluster deletion workflows.
 
+## View Clusters page
+
+On [signing in to the Console](cockroachcloud-create-your-account.html#sign-in), the **Clusters** page is displayed. The **Clusters** page provides a high-level view of your clusters.
+
+For each cluster, the following details display:
+
+- The cluster's **Name**
+- The **Version** of CockroachDB the cluster is running
+- The **Regions** the cluster is in
+- The date and time the cluster was **Created**
+
+To view and manage a specific cluster, click the name of the cluster. The [**Overview**](#view-cluster-overview) page will display.
+
+## Access tab
+
+On the **Clusters > Access** tab, Console administrators can manage Console access settings for the organization.
```

Comment:
Below, it seems Organization is capitalized. Not sure which is the style, but it should be consistent.